src/pyflex/config.py

```diff
@@ -264,6 +267,13 @@ def __init__(self, min_period, max_period, stalta_waterlevel=0.07,
                 "'interval_scheduling' or 'merge'.")
         self.resolution_strategy = resolution_strategy.lower()
 
+        if select_mode.lower() not in ["all_wave", "body_and_surface", "body_wave",
+                "surface_wave", "mantle_wave"]:
```

Comment:
I guess it makes sense to either add the `_wave` suffix to `body_and_surface` or remove it from all others. I would also rename the argument to `selection_mode`.